redis/endpoint.bal

```diff
@@ -16,26 +16,32 @@
 
 import ballerina/jballerina.java;
 
-# Represents Redis client endpoint.
+# Ballerina Redis connector provides the capability to access Redis cache.
+# This connector lets you to perform operations to access and manipulate key-value data stored in a Redis database. 
```

Comment:
Suggestion:
This connector allows you to perform operations to access and manipulate key-value data stored in a Redis database.

Reply:
Fixed